pkg/api/helpers.go

```diff
@@ -154,6 +154,17 @@ func IsStandardQuotaResourceName(str string) bool {
 	return standardQuotaResources.Has(str)
 }
 
+// AddStandardQuotaResourceNames add the resource names into the list of
+// resources known to the quota tracking system
+func AddStandardQuotaResourceNames(names ...ResourceName) {
```

Comment:
@deads2k I removed the validation from there and will open a follow up to do proper validation in validation package.